aea/registries/base.py

```diff
@@ -47,7 +46,7 @@
 Item = TypeVar("Item")
 ItemId = TypeVar("ItemId")
 ComponentId = Tuple[SkillId, str]
-SkillComponentType = TypeVar("SkillComponentType", Handler, Behaviour, Task, Model)
```

Comment:
Why is task no longer considered one?

Reply:
Because it's not properly a skill component type. In fact, we don't include tasks in configuration files, nor in any registry.